src/App.vue

```diff
@@ -134,6 +135,11 @@ export default {
     isDark(val) {
       this.$vuetify.theme.dark = val
     },
+    refetchAgents(val) {
+      if (val) {
+        this.$apollo.queries.agents.refresh()
+      }
+    },
```

Comment:
I'm a little confused what this watcher is doing that `this.$globalApolloQueries['agents'].refetch()` wouldn't do without a new store pattern